lib/src/lifecycle_module.dart

```diff
@@ -334,8 +334,14 @@ abstract class LifecycleModule extends SimpleModule
         _didLoadChildModuleController.add(childModule);
         completer.complete();
       } catch (error, stackTrace) {
-        await _didUnloadChildModuleSubscriptions[childModule]?.cancel();
-        await _willUnloadChildModuleSubscriptions[childModule]?.cancel();
+        StreamSubscription<LifecycleModule> didUnloadSub =
+            _didUnloadChildModuleSubscriptions.remove(childModule);
+        await didUnloadSub?.cancel();
```

Comment:
If there is any preference to had, it would be consistency. Would you mind updating the code below to maximize readability?